web/models/setting/settings.ex

```diff
@@ -25,6 +25,18 @@ defmodule CoursePlanner.Settings do
     Repo.all(@all_query)
   end
 
+  def get_value(name) do
```

Comment:
it is a nice interface to receive a default value which will be returned when `system_variable` matches to nil on https://github.com/digitalnatives/course_planner/pull/140/files#diff-46197ed733140459d99a5c8b132185ffR32 just like `Map.get` interface

```elixir
def get_value(name, default \\ nil) do
...
      nil -> {:ok, default}
...
```